packages/ibm-products/src/components/SidePanel/SidePanel.js

```diff
@@ -393,23 +367,19 @@ export let SidePanel = React.forwardRef(
       open,
     ]);
 
-    // adds focus trap functionality
-    /* istanbul ignore next */
-    const handleBlur = ({
-      target: oldActiveNode,
-      relatedTarget: currentActiveNode,
-    }) => {
-      // focus trap should only be set if the side panel is a `slideOver` type
-      if (open && innerContentRef && !slideIn) {
-        wrapFocus({
-          bodyNode: innerContentRef.current,
-          startTrapRef,
-          endTrapRef,
-          currentActiveNode,
-          oldActiveNode,
-        });
+    useEffect(() => {
+      if (open) {
+        setTimeout(() => {
+          const primeFocusEl = document.querySelector(selectorPrimaryFocus);
```

Comment:
We should add a check here to make sure `document` is defined (ie in the case of SSR apps like next.js).

Reply:
Done.